src/ui/settings/ViewSubscriptionManagement/ViewSubscriptionManagementStates.qml

```diff
@@ -0,0 +1,149 @@
+            }
+        },
+
+        State {
+            name: "StateUnblockCodeNeeded"
+            when: (
+                VPNAuthInApp.state === VPNAuthInApp.StateUnblockCodeNeeded
+                || VPNAuthInApp.state === VPNAuthInApp.StateVerifyingUnblockCode
+            )
+            PropertyChanges {
+                target: loader
+                source: "qrc:/ui/authenticationInApp/ViewAuthenticationUnblockCodeNeeded.qml"
+            }
+        },
+
+        State {
+            name: "StateVerificationSessionByEmailNeeded"
+            when: (
+                VPNAuthInApp.state === VPNAuthInApp.StateVerificationSessionByEmailNeeded
+                || VPNAuthInApp.state === VPNAuthInApp.StateVerifyingSessionEmailCode
+            )
+            PropertyChanges {
+                target: loader
+                source: "qrc:/ui/authenticationInApp/ViewAuthenticationVerificationSessionByEmailNeeded.qml"
+            }
+        },
+
+        State {
+            name: "StateVerificationSessionByTotpNeeded"
+            when: (
+                VPNAuthInApp.state === VPNAuthInApp.StateVerificationSessionByTotpNeeded
+                || VPNAuthInApp.state === VPNAuthInApp.StateVerifyingSessionTotpCode
+            )
+            PropertyChanges {
+                target: loader
+                source: "qrc:/ui/authenticationInApp/ViewAuthenticationVerificationSessionByTotpNeeded.qml"
+            }
+        },
+
+        State {
+            name: "StateSubscriptionManagement"
+            when: (
+                VPNAuthInApp.state === VPNAuthInApp.StateInitializing
+                || VPNAuthInApp.state === VPNAuthInApp.StateAuthenticated
+            ) && VPNSubscriptionData.initialized
+            PropertyChanges {
+                target: loader
+                source: "qrc:/ui/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml"
+            }
+        }
+    ]
+
+    Connections {
+        target: VPN
+
+        // TODO: Remove example data
+        function onSubscriptionManagementNeeded() {
+            const exampleData = '{
+                "created_at": 1626704467,
+                "expires_on": 1652970067,
+                "is_cancelled": false,
+                "payment": {
+                    "credit_card_brand": "visa",
+                    "credit_card_exp_month": 12,
+                    "credit_card_exp_year": 2022,
+                    "credit_card_last4": "0016",
+                    "provider": "stripe",
+                    "type": "credit"
+                },
+                "plan": {
+                    "amount": 499,
+                    "currency": "eur",
+                    "interval_count": 1,
+                    "interval": "month"
+                },
+                "status": "active",
+                "type": "web"
+            }';
+            VPNSubscriptionData.fromJson(exampleData);
+        }
+    }
```

Comment:
Only for testing purposes until the Guardian endpoint `/subscriptionDetails` is available that is introduced with PR #1502.